src/functional/iterator/__init__.py

```diff
@@ -1,13 +1,10 @@
 from typing import Optional, Union
 
-from . import builtins, runtime, tracing
-
-
-__all__ = ["builtins", "runtime", "tracing"]
```

Comment:
If you don't want `from iterator import *` to import anything use `__all__ = []`. Removing it entirely will import everything without an underscore (see https://stackoverflow.com/questions/44834/what-does-all-mean-in-python).